docs/porting/target/bootstrap.md

```diff
@@ -28,7 +28,7 @@ If you are updating your own linker script, you must:
     - Arm - The boot stack is the `ARM_LIB_STACK` region.
     - GCC_ARM - The boot stack starts at the symbol `__StackLimit` and ends at the symbol `__StackTop`.
 - Add defines for a relocatable application - `MBED_APP_START` and `MBED_APP_SIZE`.
-- Add the define for boot stack size - `MBED_BOOT_STACK_SIZE`.
+- Add the define for boot stack size - `MBED_CONF_TARGET_BOOT_STACK_SIZE`.
```

Comment:
Maybe, indicate that define has to be define in rtos json lib config ?

Reply:
I am not sure this is required to specified in the Mbed config. The linkerscript can specify a fallback value. I may be wrong about this being optional, though.